DNN Platform/Library/Common/Initialize.cs

```diff
@@ -383,11 +383,17 @@ private static Version GetNETFrameworkVersion()
             // Otherwise utilize release DWORD from registry to determine version
             // Reference List: https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies
             var release = GetDotNet4ReleaseNumberFromRegistry();
+            if (release >= 533320)
+            {
+                version = "4.8.1";
+            }
+
             if (release >= 528040)

```

Comment:
Does this actually work? Doesn't this need to be in descending order? If `release` is 533320, that should mean 4.8.1, but then it will _also_ be greater than 528040 and 461808, so won't `version` end up being set to `"4.7.2"`? Maybe I'm getting this mixed up in my head, but it doesn't read to me like it should work correctly without an `else` before each `if`.

Reply:
Good catch, it doesn't. It was lazy-converted from using `return "4.8";`. I'll switch this pull to draft until I can get this done properly. Thanks!

Reply:
Logic is correct now. Humbly, thanks!